src/jsutils/__tests__/Path-test.js

```diff
@@ -0,0 +1,25 @@
+// @flow strict
+
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+
+import { addPath, pathToArray } from '../Path';
+import { type Path } from '../Path';
+
+describe('Path', () => {
+  it('can add a new key to an existing Path', () => {
+    const first: Path = { prev: undefined, key: 1 };
+    const second = addPath(first, 'two');
+    expect(second.prev).to.equal(first);
```

Comment:
```suggestion
    expect(second.prev).to.deep.equal(first);
```

Reply:
Actually I want the objects to be strictly equal in this case, that's why I used `.equal`.

Reply:
Looks good to me :+1: